string inp_name
---
string name
float32 num
